packages/create-jest/src/runCreate.ts

```diff
@@ -28,9 +31,36 @@ const {
 
 const getConfigFilename = (ext: string) => JEST_CONFIG_BASE_NAME + ext;
 
-export default async function init(
-  rootDir: string = tryRealpath(process.cwd()),
+export async function runCLI(): Promise<void> {
+  try {
+    const argv = await yargs(process.argv.slice(2))
+      .usage('Usage: $0 [rootDir]')
+      .version(getVersion())
+      .alias('help', 'h')
+      .epilogue('Documentation: https://jestjs.io/').argv;
+
+    const rootDir =
+      typeof argv._[0] !== 'undefined' ? String(argv._[0]) : undefined;
+
+    await runCreate(rootDir);
+  } catch (error: unknown) {
+    clearLine(process.stderr);
+    clearLine(process.stdout);
+    if (error instanceof Error && Boolean(error?.stack)) {
+      console.error(chalk.red(error.stack));
+    } else {
+      console.error(chalk.red(error));
+    }
+
+    exit(1);
+    throw error;
+  }
+}
+
+export async function runCreate(
+  rootDir: string = process.cwd(),
 ): Promise<void> {
+  rootDir = tryRealpath(rootDir);
   // prerequisite checks
   const projectPackageJsonPath: string = path.join(rootDir, PACKAGE_JSON);
```

Comment:
Interesting why ESLint did not catch this:

```suggestion
  const projectPackageJsonPath = path.join(rootDir, PACKAGE_JSON);
```